spark-connector/spark-common/src/main/java/org/apache/gravitino/spark/connector/iceberg/GravitinoIcebergCatalog.java

```diff
@@ -58,6 +60,16 @@ public class GravitinoIcebergCatalog extends BaseCatalog
   @Override
   protected TableCatalog createAndInitSparkCatalog(
       String name, CaseInsensitiveStringMap options, Map<String, String> properties) {
+    String jdbcDriver = properties.get(IcebergConstants.GRAVITINO_JDBC_DRIVER);
+    if (StringUtils.isNotBlank(jdbcDriver)) {
+      // If `spark.sql.hive.metastore.jars` is set to path, Spark will use IsolatedClientLoader to
```

Comment:
-> ...If `spark.sql.hive.metastore.jars` is set, (Remove to path or use something other to place words `to path`)

Reply:
updated